Automattic/wp-calypso
# Move Sharing to Marketing

#### Changes proposed in this Pull Request

* Removes sharing and creates a marketing item in the sidebar

#### Testing instructions

* Observe marketing item under the tools menu
* Click it observe page loads and clicking between tabs works
* Go directly to /sharing/{ siteName }, observe it redirects to /marketing/connections
* Go directly to /sharing/buttons/{ siteName }, observe it redirects to /marketing/sharing-buttons
* Turn off `"ui/streamlined-nav-drawer": true,` in `config/development.json`
* Restart `npm start`
* observe old menu
* Follow same testing above
